pkg/driver/kubernetes_driver_test.go

```diff
@@ -0,0 +1,47 @@
+package driver
+
+// TODO(jlegrone): uncomment tests when k8s.io/client-go/kubernetes/fake is available
```

Comment:
We need to update k8s.io/client-go in order to get the fake client implementations.

Reply:
I think you should generate a fake (somehow) and check it in to this repo (possibly as a temporary measure until the above fake client is available) so that a good unit test case be provided in this PR. Otherwise, it's likely that the code under test won't be properly structured for unit testing and will therefore have poor coverage.